### ISSUE TITLE
False positive for "blank line" missing after a function declaration

### ISSUE THREAD

**Issue by elenadimitrova** [duaraghav8/Solium#35]
This wasn't happening before v1.6 but now it is. The code below will give me a `FunctionDeclaration must be succeeded by 1 blank line` 

```
/// @notice this function returns something important
/// @return description
function someFunction() constant returns(uint256) {
    return 1;
}

  /// @notice gets something else important
  function anotherFunction() constant returns(uint256)
  {
    return 2;
  }
```

**Comment by elenadimitrova** [comment]
Basically seems to be due to the comments section which somehow confuse it. If comments are removed, then the rule passes.

**Comment by duaraghav8** [comment]
Solved in 0.2.1, thanks to @cgewecke
